elements/pfe-tabs/src/pfe-tabs.js

```diff
@@ -167,15 +168,15 @@ class PfeTabs extends PFElement {
   }
 
   connectedCallback() {
-    super.connectedCallback();
-
-    this.addEventListener("keydown", this._onKeyDown);
-    this.addEventListener("click", this._onClick);
-
     Promise.all([customElements.whenDefined(PfeTab.tag), customElements.whenDefined(PfeTabPanel.tag)]).then(() => {
+      super.connectedCallback();
```

Comment:
Wrapped everything in the connectedCallback() in a Promise to wait for the dependent components to upgrade first.